### DISCUSSION TITLE
如何在Alist V3中添加ICP备案？

### DISCUSSION THREAD

**Discussion by zebwqfox** [AlistGo/alist#2756] [Q&A] [answered]
Alist V3中似乎取消了后端直接设置ICP备案号的功能，想问一下如何通过自定义body的方式实现展示ICP备案号并链接到beian.miit.gov.cn呢，非常需要这个功能

**Comment by mtk2025** [comment, marked as answer]
https://blog.csdn.net/u012514495/article/details/125772285
可以去看看，有常规代码部署

**Reply by zebwqfox** [reply]
好的，谢谢你~
